tests/test_ingest_files.py

```diff
@@ -0,0 +1,95 @@
+"""Test ingest_files.py
+
+These tests verifies:
+    - Group type annotations that have numeric-like values are being treated as strings
+    - Numeric columns are rounded to 3 decimals points
+    - Filtering cell names (given from cluster file) in metadata correctly
+
+PREREQUISITES
+Spin up Python 3.6 virtualenv, install Python dependencies in requirements.txt
+and Firestore emulator must be running, see PR26 for instructions
+(https://github.com/broadinstitute/scp-ingest-pipeline/pull/26)
```

Comment:
Is a running Firestore emulator truly a prerequisite here?  These tests seem independent from Firestore.

Reply:
You're right. Even with copy and paste, reading is essential!